### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by falinsky**
Remove incorrectly used article

### CODE REVIEW

File: packages/documentation/copy/en/handbook-v2/More on Functions.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ const longerString = longest("alice", "bob");
 const notOK = longest(10, 100);
 ```
 
-There are a interesting few things to note in this example.
+There are interesting few things to note in this example.
```
Comment (orta) [inline packages/documentation/copy/en/handbook-v2/More on Functions.md]:
```suggestion
There are few interesting things to note in this example.
```

**Comment by orta** [comment]
Good spot - I've also tweaked it 👍🏻

**Comment by falinsky** [comment]
@orta thanks for the suggestion. It's applied. Is it ready to be merged now?

**Comment by orta** [comment]
Yep, was just waiting for it to green 👍🏻

**Comment by falinsky** [comment]
@orta thanks 👍

**Pull request merged**